catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/event/impl/SubscriptionImpl.java

```diff
@@ -22,13 +22,15 @@
 import ddf.catalog.event.Subscription;
 
 public class SubscriptionImpl implements Subscription {
-    private Filter filter;
+    protected Filter filter;
 
-    private DeliveryMethod dm;
+    protected DeliveryMethod dm;
 
-    private Set<String> sourceIds;
+    protected Set<String> sourceIds;
 
-    private boolean enterprise;
+    protected boolean enterprise;
+
+    protected boolean active = true;
```

Comment:
Do we expect clients to change this flag after the class has been instantiated? If not, should we just add it to the constructor and make it `private`? If we expect it to be changed, could we have a setter for it instead of making it `protected`, or make the class immutable and have clients create a new instance if they want to change something?